Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -332,12 +334,29 @@ class MainTabBarViewController: UITabBarController {
     
     private func showTransactionDetails(safeTxHash: Data) {
         let vc = ViewControllerFactory.transactionDetailsViewController(safeTxHash: safeTxHash)
-        present(vc, animated: true, completion: nil)
+        presentWhenReady(vc)
     }
     
     private func showTransactionDetails(transactionDetails: SCGModels.TransactionDetails) {
         let vc = ViewControllerFactory.transactionDetailsViewController(transaction: transactionDetails)
-        present(vc, animated: true, completion: nil)
+        presentWhenReady(vc)
+    }
+
+    private var delayedPresentationViewController: UIViewController?
+
+    private func presentWhenReady(_ vc: UIViewController) {
+        if view.window == nil {
+            delayedPresentationViewController = vc
```

Comment:
how is it possible? If app is closed view.widow == nil, but thew view controller is not nil?